src/animation/panel-animation.js

```diff
@@ -67,8 +67,8 @@ export default class PanelAnimation {
    * Usually it isn't necessary to override this
    */
   playNextFrame() {
-    this.currentFrame++;
-    
+    this.currentFrame = this.currentFrame + 1;
```

Comment:
It's against out coding style: https://github.com/anyWareSculpture/coding-style/blob/master/CODING_STYLE.md#operators